### PR TITLE
Devops Daycamp 2015 beginner track update

### PR DESCRIPTION
**Pull request by ElijahCaine**
This is not quite finished but should point us in the right direction. Feedback is encouraged. If you want to make changes after discussing push to the branch as I can't guarantee I'll be able to respond and make changes for the next few weeks.

@dspt @jordane @lucywyman @iankronquist @Phillels @ramereth 

The repository I am going to use for the git walk-through will be a simple flask app, probably *[this gist](https://gist.github.com/ElijahCaine/186aa98933ab41b413bd).*

**NOTE** This has a lot of changes that are unrelated to the restructuring of the daycamp beginner track slides, those unrelated changes were because sphinx was complaining about the build process, those problems are now fixed thankfully.

I'm not sure how it was compiling before...

### CODE REVIEW

File: source/11_tools_debuggers.rst [hunk 1]
```diff
@@ -192,10 +192,11 @@ Type Checking
 .. code-block:: bash
 
 	parser.c:53:31: warning: incompatible pointer types passing
-	'struct token *' to parameter of type 'struct token_list *'
+	'struct token *' to parameter of type 'struct token_list'*
```
Comment (lucywyman) [inline source/11_tools_debuggers.rst]: Should this be 'struct token_list*' ? With the asterisk inside the single-quote?

File: source/daycamp_introtrack_session1.rst [hunk 2]
```diff
@@ -176,20 +185,17 @@ Essential Vocabulary
...
+* Host
+* Guest
+* Virtual Machine (VM)
 * terminal/command line/cli
 
 Operating System
 ----------------
 
-.. note:: jme
+.. note:: jme (these are words)
```
Comment (lucywyman) [inline source/daycamp_introtrack_session1.rst]: Do we need this?

File: source/daycamp_introtrack_session2.rst [hunk 3]
```diff
@@ -516,404 +313,202 @@ Types of files
...
-* Install packages from a central repository
-* Search for information on installed packages and files
-* Pre-built binaries (usually)
-* Find out which package provides a required library or file
-
-**Popular Linux Package Managers**
-* .deb / APT + dpkg (used by Debian, Ubuntu, Linux Mint)
-* .rpm / YUM + rpm (used by RedHat, CentOS, Fedora)
 
 .. RPM & yum (RedHat, CentOS, Fedora)
```
Comment (lucywyman) [inline source/daycamp_introtrack_session2.rst]: I don't recall why this is commented out -- should we uncomment or remove it?

**Comment by dspt** [comment]
Fixed @lucywyman 's suggestions.  As far as the test app goes, what would be good changes to make that would teach version control and text editing (in vim?)?

**Comment by lucywyman** [comment]
I think one year we just had a student repository, where people would add a file with their name and some information about them.  We could do something similar, but maybe more fun (incentives?), and maybe less personal (not really sure what this would be either...)?   I think keeping the actual content of the activity simple is a good idea, since the point of the exercise isn't writing $language, which is why just plain text seems good, but there are certainly other possibilities.

@dspt Should that go in a separate PR?  Totally fine if it belongs here, I just thought this was mostly spelling/grammatic/style fix-ups.

**Comment by lucywyman** [comment]
+1 on this

**Pull request merged**